start-splice-cluster

```diff
@@ -48,7 +48,7 @@ wait_for () {
 }
 
 BASE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-DEFAULT_PROFILE="hdp3.1.0"  # default hbase platform profile
+DEFAULT_PROFILE="cdh6.3.0"  # default hbase platform profile
```

Comment:
Why?

Reply:
We use cdh more often and it's a pain to switch profiles back and forth in intellij, if people don't mind I'd rather have it be the default profile.